tests/test_infer.py

```diff
@@ -25,9 +25,9 @@ def test_upperlimit(tmpdir, hypotest_args):
     Check that the default return structure of pyhf.infer.hypotest is as expected
     """
     _, data, model = hypotest_args
-    observed_limit, expected_limits = pyhf.infer.intervals.upperlimit(
-        data, model, scan=np.linspace(0, 5, 11)
-    )
+    results = pyhf.infer.intervals.upperlimit(data, model, scan=np.linspace(0, 5, 11))
```

Comment:
...not really sure why this is complaining, but i think it is fine.

Reply:
it keeps complaining. codefactor bug.